### ISSUE TITLE
Problem connecting to Processmaker server

### ISSUE THREAD

**Issue by BlackxRyan** [tomolimo/processmaker#56]
Hi.

I have a Debian 8 and I installed GLPI (9.2.2) and Processmaker (3.2.1) on my machine. They both work correctly.

I followed the documentation to install Processmaker : [](http://wiki.processmaker.com/3.0/ProcessMaker_Debian_Installation)

Now I must connect Processmaker to GLPI. So I downloaded the plugin, entered my informations and I have the message "Test failed stdClass Object ( )". 

![image](https://user-images.githubusercontent.com/36700904/39126937-e427830a-4703-11e8-8ae5-3add436ec5c3.png)

I am sure the login and the password are the correct ones. 

10.0.2.15 is the local IP of my Debian, I'm using the port 8080 to access Processmaker and I use the port 80 for GLPI.

Thanks in advance.

**Comment by tomolimo** [comment]
Hello @BlackxRyan ,
As written here https://github.com/tomolimo/processmaker-server/wiki/ProcessMaker-server-installation#notes, you can only use the ProcessMaker released by us: the latest release is here: https://github.com/tomolimo/processmaker-server/releases/latest
Sorry for that, but currently, the plugin is only compatible with the 3.0.1.8-RE-1.8 release
Thank you
Regards,
Tomolimo

**Comment by BlackxRyan** [comment]
Ok, thanks for the fast answer !

**Comment by tomolimo** [comment]
You're welcome, keep me informed

**Comment by BlackxRyan** [comment]
I'll tell you if it works :)

**Comment by tomolimo** [comment]
Hello @BlackxRyan ,
The version of ProcessMaker we are delivering has been modified (and sometimes fixed) to be compatible with GLPI, so you MUST use this release, that's not an option.
Sorry for that,
Thank you
Regards,
Tomolimo

**Comment by BlackxRyan** [comment]
No problem, I'll install this version and I'll tell you if it works :)

**Comment by BlackxRyan** [comment]
Ok, it works perfectly but I still have the problem i mentioned. It still show the processes which don't exist anymore.

**Comment by tomolimo** [comment]
Yes, they are not cleared from the list, it's in my long list of evolutions :)
To clean the not already used processes (=no cases yet) from the process list when they no longer exist in PM server.

**Comment by BlackxRyan** [comment]
Ok I see, do you know where these processes are saved in the plugin files ?

**Comment by tomolimo** [comment]
You're welcome, and don't hesitate if you have any issue.
regards,
Tomolimo
